### PR TITLE
Clean

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️
| Tests pass?   | ❌

I want to divide the routine and useful work into different PR. It will help to concentrate both in the development and in the review code. 

So I did most of the routine work in this PR.

### CODE REVIEW

**Comment by xepozz** [comment]
Needs to add the @yiisoft/reviewers to this repo.

File: composer.json [hunk 1]
```diff
@@ -21,6 +21,8 @@
         "yiisoft/yii-dataview": "^3.0@dev",
         "yiisoft/arrays": "^3.0@dev",
         "yiisoft/strings": "^3.0@dev",
```
Comment (samdark) [inline composer.json]: Not used at all:

- yiisoft/strings
- yiisoft/yii-dataview
- yiisoft/di
- yiisoft/event-dispatcher

Should be moved to require from require-dev:

- yiisoft/log
- yiisoft/view

File: src/Actions/DB/ExplainAction.php [hunk 2]
```diff
@@ -1,42 +1,12 @@
...
-/**
- * ExplainAction provides EXPLAIN information for SQL queries
- *
- * @property \Yiisoft\Yii\Debug\Controllers\DefaultController|\yii\web\Controller|\Yiisoft\Yii\Console\Controller $controller the controller that owns this action
- *
- * @author Laszlo <github@lvlconsultancy.nl>
- * @since 2.0.6
- */
-class ExplainAction extends Action
+class ExplainAction
```
Comment (samdark) [inline src/Actions/DB/ExplainAction.php]: I think the description of the purpose of the class was helpful.

**Comment by samdark** [comment]
Other than that it is OK as preliminary cleanup.

File: composer.json (hunk 1 above)
Comment (xepozz) [reply composer.json]: I will remove unused dependencies later

File: src/Actions/DB/ExplainAction.php (hunk 2 above)
Comment (xepozz) [reply src/Actions/DB/ExplainAction.php]: Ok I will restore it

**Comment by samdark** [comment]
Thanks!

**Pull request merged**